core/arch/arm/kernel/thread.c

```diff
@@ -114,6 +114,17 @@ struct thread_core_local thread_core_local[CFG_TEE_CORE_NB_CORE] __nex_bss;
 #define STACK_CHECK_EXTRA	0
 #endif
 
+/*
+ * Extra space can be added to the TMP and THREAD stacks when the defaults are
+ * not sufficient for certain configurations
+ */
+#ifndef CFG_STACK_TMP_EXTRA
+#define CFG_STACK_TMP_EXTRA	0
+#endif
+#ifndef CFG_STACK_THREAD_EXTRA
+#define CFG_STACK_THREAD_EXTRA	0
+#endif
```

Comment:
Sorry, I missed that. `CFG_*` deafult value should be set from the build configuration, not in the code itselft.
Please move this to **mk/config.mk** as:
```
# Extra space can be added to the TMP and THREAD stacks when the defaults are
# not sufficient for certain configurations.
CFG_STACK_TMP_EXTRA ?= 0
CFG_STACK_THREAD_EXTRA ?= 0

Reply:
ok